snapcraft.yaml

```diff
@@ -0,0 +1,28 @@
+name: keepassxc
+version: develop
+summary: community driven port of the windows application “Keepass Password Safe”
+description: |
+  KeePassXC is an application for people with extremly high demands on secure
+  personal data management. It has a light interface, is cross platform and
+  published under the terms of the GNU General Public License.
+confinement: strict
+
+apps:
+  keepassxc:
+    command: desktop-launch keepassxc
+    plugs: [unity7, opengl, gsettings, home]
+
+parts:
+  keepassx:
+    source: .
+    plugin: cmake
+    build-packages:
+      - g++
+      - libgcrypt20-dev
+      - libmicrohttpd-dev
+      - libqt4-dev # upstream cmake needs libqt4-dev to find qt5-core
```

Comment:
oh, I did this so long ago that now I don't remember why this weird line. Thanks for helping to remove it.